### PR TITLE
build: Update build version to v1.6.8 for the release

### PR DESCRIPTION
**Pull request by travisn**
**Description of your changes:**
With the patch release the version is updated in the manifests and examples

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

[skip ci]

### CODE REVIEW

File: Documentation/ceph-common-issues.md [hunk 1]
```diff
@@ -958,7 +958,7 @@ You need to create the CRDs found in `cluster/examples/kubernetes/ceph/pre-k8s-1
 ## Unexpected partitions created
 
 ### Symptoms
-**Users running Rook versions v1.6.0-v1.6.7 may observe unwanted OSDs on partitions that appear
+**Users running Rook versions v1.6.0-v1.6.8 may observe unwanted OSDs on partitions that appear
```
Comment (BlaineEXE) [inline Documentation/ceph-common-issues.md:961]: Should stay the same I think.

File: Documentation/ceph-common-issues.md [hunk 2]
```diff
@@ -993,7 +993,7 @@ sdd      8:48   0     3T  0 disk
 You can see https://github.com/rook/rook/issues/7940 for more detailed information and discussion.
 
 ### Solution
-#### Recover from corruption (v1.6.0-v1.6.7)
+#### Recover from corruption (v1.6.0-v1.6.8)
```
Comment (BlaineEXE) [inline Documentation/ceph-common-issues.md:996]: Should stay the same I think.

**Review by BlaineEXE — CHANGES_REQUESTED** [review]

File: Documentation/ceph-common-issues.md [hunk 3]
```diff
@@ -31,6 +31,7 @@ If after trying the suggestions found on this page and the problem is not resolv
 * [LVM metadata can be corrupted with OSD on LV-backed PVC](#lvm-metadata-can-be-corrupted-with-osd-on-lv-backed-pvc)
 * [OSD prepare job fails due to low aio-max-nr setting](#osd-prepare-job-fails-due-to-low-aio-max-nr-setting)
 * [Failed to create CRDs](#failed-to-create-crds)
+* [Unexpected partitions created](#unexpected-partitions-created)
```
Comment (BlaineEXE) [inline Documentation/ceph-common-issues.md:34]: Thanks. I must have missed this.

**Review by BlaineEXE — APPROVED** [review]
LGTM. Thanks for updating the TOC in the common issues.

File: Documentation/ceph-common-issues.md (hunk 3 above)
Comment (travisn) [reply Documentation/ceph-common-issues.md:34]: It was auto-generated by vscode, lucky me for touching that file :)

**Pull request merged**